src/effects/effectsmanager.cpp

```diff
@@ -498,3 +496,32 @@ void EffectsManager::collectGarbage(const EffectsRequest* pRequest) {
                 pRequest->DisableInputChannelForChain.pChannelHandle);
     }
 }
+
+void EffectsManager::loadEffectChainPresets() {
+    QDir settingsPath(m_pConfig->getSettingsPath());
+    QFile file(settingsPath.absoluteFilePath("effects.xml"));
+    QDomDocument doc;
+
+    if (!file.open(QIODevice::ReadOnly)) {
+        return;
+    } else if (!doc.setContent(&file)) {
+        file.close();
+        return;
+    }
+    file.close();
+
+    QDomElement root = doc.documentElement();
+    QDomElement rackElement = XmlParse::selectElement(root, EffectXml::Rack);
```

Comment:
Add a comment explaining that this element is superfluous but it is kept for backwards compatibility.